pages/cars.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+ <h1> {{ data.cars }} </h1>
+</template>
+
+<script setup>
+const { data } = await useFetch('/api/cars')
```

Comment:
how this suppose to work?

Reply:
It probably won't work on Netlify, cuz it's using Nuxt Server.

Reply:
oh ok, i will try to do it on the page then